nixos/modules/services/misc/podgrab.nix

```diff
@@ -0,0 +1,69 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.podgrab;
+in
+{
+  options.services.podgrab = with lib; {
+    enable = mkEnableOption "Podgrab, a self-hosted podcast manager";
+
+    user = mkOption {
+      type = types.str;
+      default = "podgrab";
+      description = "User account under which podgrab runs.";
+    };
+
```

Comment:
It might be convenient to have a port option here as well, which I think needs to set the PORT environment variable for Gin to pick it up.

Reply:
(for reference, this comes from https://github.com/gin-gonic/gin/blob/v1.7.1/utils.go#L142)

Reply:
Ah I did not know that, I didn't see a mention of `PORT` in the project's README. Will try it out and add a test for it.

If I add a port option, I figure we don't need an `openFirewall` option.